doc/dev/installation.rst

```diff
@@ -87,6 +87,19 @@ They can be installed on Debian/Ubuntu via:
 
   sudo apt install clang lld ccache libomp-dev ninja-build make cmake
 
+They can be installed on macOS via:
+
+.. code-block:: console
+
+  brew install cmake ninja
+
+If you install Catalyst on a macOS system with ``ARM`` architecture, you need to install
+the ``llvm-tools-preview`` rustup component:
```

Comment:
Actually, this might be even better
```suggestion
If you install Catalyst on a macOS system with ``ARM`` architecture (e.g. Apple M1/M2), you 
additionally need to install the ``llvm-tools-preview`` rustup component:
```